react/features/base/connection/actions.any.ts

```diff
@@ -72,7 +72,7 @@ export type ConnectionFailedError = {
  *     connection: JitsiConnection
  * }}
  */
-export function connectionDisconnected(connection: Object) {
+export function connectionDisconnected(connection?: Object) {
```

Comment:
when would we dispatch this without a connection?

Reply:
This is necessary because we dispatch with the connection object we get from the store which is of type `Object | undefined`

Reply:
Ah, bummer.